backend/tests/regression/answer_quality/launch_search_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export PYTHONPATH="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../../.." && pwd):$PYTHONPATH"
+export MODEL_SERVER_HOST="18.218.165.17" 
```

Comment:
not as bad as hardcoding API keys and as discussed, the worst outcome is a denial of service on our testing which is basically harmless but... for good practice we should not hardcode IPs